jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -137,7 +137,7 @@ def test_list_mounts(pachyderm_resources, dev_server):
             "branch",
             "project",
             "commit",
-            "files",
+            "path",
```

Comment:
Does this change come from another commit?    It is actually unrelated to this change, right?
Just fixes tests?

Reply:
That's correct. It's an unrelated change that fixes the test that didn't trigger in a different commit.

Reply:
this change was required to address circle ci test failure based on feedback from @smalyala, build will not pass otherwise.